vm/src/obj/objtype.rs

```diff
@@ -26,19 +26,35 @@ impl PyValue for PyClass {
     }
 }
 
+impl IdProtocol for PyClassRef {
```

Comment:
Note: in the future we shouldn't need this impl, once `PyObjectRef` is just an alias for `PyRef<dyn PyValue>` we can implement it broadly for all refs.